src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,60 @@
+.navigation {
+  &__list {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    align-items: center;
+  }
+
+  &__item {
+    display: inline-block;
+    line-height: 60px;
+    box-sizing: border-box;
+    text-transform: uppercase;
+    color: $black;
+    list-style: none;
+    margin: 0 20px 0 0;
+    padding: 0;
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    font-family: "Roboto", sans-serif;
+    position: relative;
+    height: 100%;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 60px;
+    text-transform: uppercase;
+    color: $darcBlue;
+    text-decoration: none;
+    transition: color 0.5s;
+
+    &:hover {
+      color: $blue;
+    }
+
+    &.is-active::after {
```

Comment:
Let's create modifier for link, and move these styles to those modifier
And leave is-active class only in html for tests passing